app/webpacker/controllers/dropdown_controller.js

```diff
@@ -1,44 +1,29 @@
 import { Controller } from "stimulus";
 
 export default class extends Controller {
-  static targets = ["arrow", "menu"];
 
   connect() {
-    this.collapsedClasses = this.arrowTarget.dataset.collapsedClass.split(" ");
-    this.expandedClasses = this.arrowTarget.dataset.expandedClass.split(" ");
-    this.#hide();
-    document.addEventListener("click", this.#onBodyClick.bind(this));
+    document.body.addEventListener("click", this.#close.bind(this));
+    this.element.addEventListener("click", this.#stopPropagation.bind(this));
```

Comment:
I think this one doesn't need `bind`, because the handler only refers to the event, not `this`.

Reply:
After trying removing each one, there is a need for both.